lib/taskana-core/src/test/java/acceptance/task/CallbackStateAccTest.java

```diff
@@ -207,7 +281,9 @@ void testQueriesWithCallbackState()
 
     }
 
-    private TaskImpl createTask(TaskService taskService, CallbackState callbackState) throws WorkbasketNotFoundException, ClassificationNotFoundException, NotAuthorizedException, TaskAlreadyExistException, InvalidArgumentException {
+    private TaskImpl createTask(TaskService taskService, CallbackState callbackState)
```

Comment:
unneeded formatting? Also on other files too. It does not matter, but makes your PR bigger and thus takes longer to review.

Reply:
Agreed, this was auto-formatting (using our QA IntelliJ Formatter).